zmq/types/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "cuprate-zmq-types"
+version = "0.1.0"
+edition = "2021"
+description = "Types for the ZMQ Pub/Sub API"
+license = "MIT"
+authors = ["dimalinux", "The Monero Project"]
+repository = "https://github.com/Cuprate/cuprate/tree/main/zmq/types"
+
+[dependencies]
+serde = { workspace = true, features = ["derive"] }
+hex = { workspace = true, features = ["std", "serde"] }
+cuprate-types = { path = "../../types", features = ["hex"] }
```

Comment:
I'll merge with the remaining `main` commits and make this a workspace entry after the fixes for `main` branch commit `63216aecaead915550be93236faf6aeeebf0e04f` are in.